config/cli/bookworm/debootstrap/packages

```diff
@@ -1,36 +0,0 @@
-apt-utils
-btrfs-progs
-bzip2
-ca-certificates
-console-setup
-cpio
-cron
-dbus
-dbus-user-session
-debsums
-dmsetup
-fdisk
-gnupg
-haveged
-ifupdown
-init
-initramfs-tools
-iputils-ping
```

Comment:
We have replaced `iputils-ping` with `inetutils-ping` after this pr. But `iputils-ping` is default in debian and ubuntu.

Reply:
So you mean both are installed? Change this package back?

Reply:
These two can't be installed at the same time. I think it's better to change it back. I'm going to create a pr for this.